data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,542 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	Ikiyachee gives you directions to the lounge, and you head there to wait out the interview.`
+				goto "lounge"
+			label "accompany"
+			`	"To stay with her during the interview, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
+			choice
+				`	(Wait in the lounge.)`
+					goto "directions to lounge"
+				`	(Accompany Anna during the interview.)`
+			label "stay in interview"
+			`	You choose to remain in the room. Once the first questions are asked, Anna is all too happy to begin discussing the contents of her novel in great detail.`
+			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Heliarchs' numerous questions for clarification and elaboration make the process a long one.`
+			`	As you listen, you learn that due to the Coalition's censorship and review process, Anna will have to stay for about half a year, during which she will have an apartment to live in.`
+			choice
+				`	"Do I have to stay with Anna here for half a year?"`
+					goto "half a year"
+				`	(Say nothing.)`
+			label "rest of interview"
+			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks you and Anna.`
+			`	You leave the interview room, the Heliarchs staying in the room, presumably to continue discussion with the Kimek. As the two of you walk, Anna begins to speak.`
+				goto thanks
+			label "half a year"
+			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. To return to human space, you are free to do," says the Kimek.`
+				goto "rest of interview"
+			label "lounge"
+			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table and pass the time, occasionally going to the water dispenser to hydrate yourself.`
+			`	After thirty minutes, a Kimek approaches you. Its translation device speaks. "So, the alien in our space, you are?"`
+			# kst-1 is short for [K]imek [s]mall [t]alk 1
+			choice
+				`	"I don't want to talk to you."`
+					goto "kst-1 ignore"
+				`	"Yes, I am <first> <last>. Glad to meet you."`
+				`	"Well, I'm not the only alien."`
+					goto "kst-1 not only alien"
+				`	"We call ourselves humans, not aliens."`
+					goto "kst-1 endonym"
+			label "kst-1 glad to meet you"
+			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?" "My friend Anna Fangvin is publishing a book here," you say.`
+				goto "kst-1 anna"
+			label "kst-1 not only alien"
+			`	"If more of you, here there are, then who are they?" "It's just two; my friend Anna Fangvin and I. She's publishing a book in your space," you reply.`
```

Comment:
You should probably add a paragraph break between the Kimek's words and the player's (same on later lines too).